deploy/gk-deploy

```diff
@@ -551,7 +555,12 @@ check_pods "job-name=heketi-storage-copy-job" "Completed"
 eval_output "${CLI} delete all,service,jobs,deployment,secret --selector=\"deploy-heketi\" 2>&1"
 
 if [[ "${CLI}" == *oc\ * ]]; then
-  eval_output "${CLI} process -p HEKETI_ADMIN_KEY=${ADMIN_KEY} -p HEKETI_USER_KEY=${USER_KEY} heketi | ${CLI} create -f - 2>&1"
+    oc_version = $(${CLI} version | grep "oc")
```

Comment:
instead of duplicating the code, we should detect the version once at the top and set the VAL_SWITCH accordingly, and just use that whereever we do oc process... (i.e. use `${CLI} process ${VAL_SWITCH} KEY=VAL ...`)